src/index.html

```diff
@@ -1,13 +1,352 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>Museum</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/swiper@9/swiper-bundle.min.css">
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
+
+  <body class="page__body">
+    <header class="header page__header" id="header">
+      <div class="container">
+        <div class="header__top">
+          <div class="top-action">
+            <a href="#menu" class="top-action__burger"></a>
+            <a href="#header" class="top-action__logo"></a>
+            <a href="#" class="top-action__cross"></a>
+          </div>
+        </div>
+        <div class="header__content">
+          <div class="header__title">
+            художній музей
+          </div>
+          <div class="header__button">
+            <div class="header__action">події</div>
+            <div class="button">
+              <div class="button__text">
+                квитки
+              </div>
+              <div class="button__arrow"></div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <nav class="menu page__menu" id="menu">
+      <div class="menu__container">
+        <div class="menu__left">
+          <a href="#" class="menu__cross"></a>
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#gallery" class="menu__link">
+                Галерея
+              </a>
+            </li>
+            <li class="menu__item">
+              <a href="#header" class="menu__link">
+                Про Нас
+              </a>
+            </li>
+            <li class="menu__item">
+              <a href="#events" class="menu__link">
+                Виставки та
+                <br>
+                події
+              </a>
+            </li>
+            <li class="menu__item">
+              <a href="#contact" class="menu__link">
+                Контакти
+              </a>
+            </li>
+          </ul>
+        </div>
```

Comment:
use it in all cases
<img width="1167" alt="image" src="https://user-images.githubusercontent.com/91411692/223972337-29e2b256-4329-4350-abde-2b8115a3207c.png">

```suggestion
        <div class="menu__left">
          <a href="#" class="menu__cross"></a>
          
          <ul class="menu__list">
            <li class="menu__item">
              <a href="#gallery" class="menu__link">
                Галерея
              </a>
            </li>
            
            <li class="menu__item">
              <a href="#header" class="menu__link">
                Про Нас
              </a>
            </li>
            
            <li class="menu__item">
              <a href="#events" class="menu__link">
                Виставки та
                <br>
                події
              </a>
            </li>
            
            <li class="menu__item">
              <a href="#contact" class="menu__link">
                Контакти
              </a>
            </li>
          </ul>
        </div>
        
```